sections/sprint-5/usability-test-may-18th.tex

```diff
@@ -1,11 +1,24 @@
-\section{Usability test}
+\section{Second test on May 18th}
 The second test of the system and all of its components took place on the 18th of May.
 This test was conducted as the previous test had not managed to explore the functionality of the game as described in \autoref{sec:initial-test}.
-Due to the COVID-19 pandemic, the usability test was conducted using the group members as testers, and only one person from outside the group.
+Since then we fixed the discovered bugs.
+Due to the COVID-19 pandemic, the test was conducted using the group members as testers, and only one person from outside the group.
 
 \subsection{The setup}
 The test was carried out in two phases, one inside and one outside.
-The playing field was shaped like a rectangle for both tests, much like the previous test on the 7th of May. An example of the setup can be seen on \autoref{fig:test2-indoor-setup}
+The playing field was shaped like a rectangle for both tests, much like the previous test on the 7th of May.
+An example of the setup can be seen on \autoref{fig:test2-indoor-setup}.
+
+\subsection{Method and structure}
+The test was unstructured and could be considered as a big bang test.
+Ideally we would have liked to conduct a usability test, but as the group already have expert knowledge on how to operate the program, it is not possible to test the setup and the usability of the game.
+If we could conduct a usability test it would have been structured.
+We would only have tested our part of the system, which means that we would not have made them setup Pozyx.
```

Comment:
```suggestion
Had a usability test been conducted, it would have been focused on the usage of the system as a player, meaning that a group member would have been in charge of setting up the Pozyx system and running the host computer.
```